boards/toradex/verdin_imx8mm/verdin_imx8mm_mimx8mm6_m4.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2024 Toradex
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: verdin_imx8mm/mimx8mm6/m4
+name: Toradex Verdin iMX8M Mini
+type: mcu
+arch: arm
+ram: 128
+flash: 128
```

Comment:
Does not make sense with what is in the description

Reply:
Can you clarify? It's not clear to me what needs to change.  The RAM and FLASH on these boards is configurable so I don't really know what values should be listed here.